### ISSUE TITLE
guiCreateStaticImage unable to us dxTexture as valid image

### ISSUE THREAD

**Issue by Allerek** [multitheftauto/mtasa-blue#3271]
### Describe the bug

guiCreateStaticImage is unable to take dxImage as valid image, instead it's only possible to specify the path for the image

Did not check for other gui elements.

### Steps to reproduce

1. create dx-texture using dxCreateTexture
2. Try to use it with guiCreateStaticImage

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by xLive** [comment]
It's not a bug. [guiCreateStaticImage](https://wiki.multitheftauto.com/wiki/GuiCreateStaticImage) only supports a string path to the image.
![image](https://github.com/multitheftauto/mtasa-blue/assets/30420446/4e0f0a4d-030e-481d-aaa7-3a3940e728a2)

This should be a feature request and not a _bug report_.

**Comment by Fernando-A-Rocha** [comment]
I dont think cegui needs more features. It should be replaced entirely as already discussed many times 😅

**Comment by idarrr** [comment]
> I dont think cegui needs more features. It should be replaced entirely as already discussed many times 😅

I think it would be nice to add support for dx image or raw data to cegui image, or even add support for svg image, cause last time i tried cegui image doesn't work with svg and using dxConvertPixels also doesn't work with svg texture. It's useful for creating rounded rectangle with uncertain width and height with svg (tbh idk if it's possible). Instead of replacing entire cegui which I believe require a lot of work.

**Comment by Fernando-A-Rocha** [comment]
@idarrr use dx functions in your case

**Comment by idarrr** [comment]
Yes dx functions is amazing, but imo for interactive UI with bunch of text input, it's easier with cegui since it has a lot of built in event like cursor event etc. What about scrollpane? In dx we have to use render target and handle scrolling by ourself, and afaik render target use lots of vram and produce blurry text sometimes (idk if i'm doing it wrong tho), and render target won't work if there's no vram left, most of mta players in my country is still using igpu with 64mb of vram, even if they have 1gb of shared memory available render target won't work sometimes. So "ugly" CEGUI is the best option for me now, but replacing gui window with the help of gui image for creating simple container to UI elements, CEGUI can look beautiful.
